indico/modules/events/registration/util.py

```diff
@@ -488,6 +488,27 @@ def get_event_regforms(event, user, with_registrations=False):
     return query.all()
 
 
+def get_event_regforms_registrations(event, user, include_scheduled=True):
+    """Get a tuple, which includes:
```

Comment:
First docstring line should be a concise single-sentence summary - if there's no good way to write one, wrap right after the `"""` and start in the next line (without extra indentation).

This could be a good first line here:


```python
"""Get regforms and the associated registrations for an event+user.

...
"""
```

And then explain details later (similar to how it's done in git commits with the empty line between).